CDCgov/prime-reportstream
# Experience/12974/e2e daily data updates

This PR ...

Updates e2e on Daily Data, specifically the filter component.

Test Steps:
1. Currently using non mock data so you will need to make sure that your local receiver is part of Alaska org
2. Run `./gradlew testSmoke` which will populate data for Alaska
3. Run playwright tests
`yarn run test:e2e-ui`

## Changes
- Filter specific e2e tests for daily data.
- fixed some bugs within the filter

## Checklist

### Testing

## Linked Issues
- Fixes #12974
